components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIExportUtil.java

```diff
@@ -215,6 +216,66 @@ private static void exportAPIThumbnail(String archivePath, APIIdentifier apiIden
         }
     }
 
+
+    /**
+     * Retrieve SOAP to REST mediation logic for the exporting API and store it in the archive directory
+     *
+     * @param apiIdentifier ID of the requesting API
+     * @param registry      Current tenant registry
+     * @throws APIImportExportException If an error occurs while retrieving image from the registry or
+     *                            storing in the archive directory
+     */
+    private static void exportSOAPToRESTMediation(String archivePath,APIIdentifier apiIdentifier, Registry registry)
```

Comment:
formatting issue ( space needed after the comma)

Reply:
resolved by #8332